_lessons/19-fancy-link-effects.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Fancy Link Effects
+lesson: 19
+layout: default
+---
+
+This project is coming along nicely; we have finished our layout and chosen some colors to give it some personality. The next thing we'll do is customize the sidebar links. 
+
+### Linking with Style
+
+Let's customize our links by: 
+
+- Removing the underline
+- Adding font and background colors
+- Spacing them out and padding the contents
+- Add a decorative partial border
+
+#### Remove the Default Underline
+
+Links are underlined by default, but this can be removed with CSS. Normally, underline is controlled by the `text-decoration` property, which sets decorative lines on text. You can actually control a lot with this property, but we will focus on two possible values: 
+
+- `text-decoration: underline;` specifies that the text should be underlined.
```

Comment:
We never end up mentioning `text-decoration: underline;` ever again lol.

Reply:
I think links being underlined on hover make a lot more sense than strikethrough.

Strikethrough is a lot more situational, I think. Maybe if you want to include it, you can mention it in an aside or something.